src/ansys/geometry/core/geometry/curves/line.py

```diff
@@ -28,21 +28,20 @@
 from beartype.typing import Union
 import numpy as np
 
-from ansys.geometry.core.math.matrix import Matrix44
-from ansys.geometry.core.math.point import Point3D
-from ansys.geometry.core.math.vector import UnitVector3D, Vector3D
-from ansys.geometry.core.misc.accuracy import LENGTH_ACCURACY
-from ansys.geometry.core.primitives.curve_evaluation import CurveEvaluation
-from ansys.geometry.core.primitives.parameterization import (
+from ansys.geometry.core.geometry.curves.curve import Curve
+from ansys.geometry.core.geometry.curves.curve_evaluation import CurveEvaluation
+from ansys.geometry.core.geometry.parameterization import (
     Interval,
     Parameterization,
     ParamForm,
     ParamType,
 )
-from ansys.geometry.core.typing import RealSequence
+from ansys.geometry.core.math import Matrix44, Point3D, UnitVector3D, Vector3D
+from ansys.geometry.core.misc.accuracy import LENGTH_ACCURACY
+from ansys.geometry.core.typing import Real, RealSequence
 
 
-class Line:
+class Line(Curve):
     """
     Provides 3D line representation.
```

Comment:
Here is a good example of class description that begins with a verb with an "s" ending and is written in plain English.